src/nvim/eval/funcs.c

```diff
@@ -5277,7 +5277,7 @@ static void libcall_common(typval_T *argvars, typval_T *rettv, int out_type)
   // input variables
   char *str_in = (in_type == VAR_STRING)
       ? (char *)argvars[2].vval.v_string : NULL;
-  int64_t int_in = argvars[2].vval.v_number;
+  int int_in = argvars[2].vval.v_number;
```

Comment:
wont the absence of casting from int64_t to int trigger any PVS alert ?

Reply:
No.  We do this in other places in the code, too.